<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>29</y>
      <width>721</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>701</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="3">
       <widget class="QPushButton" name="btn_exec">
        <property name="text">
         <string>execute</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="btn_exec_2">
        <property name="text">
         <string>execute</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>no acition</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>add item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>delete item</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="TYPE">
        <property name="text">
         <string>TYPE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="INPUT">
        <property name="text">
         <string>INPUT VALUE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>string</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>list</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tuple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dictionary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>variable</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="ID">
        <property name="text">
         <string>ASSIGNED ID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>VARIABLE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="TRACE">
        <property name="text">
         <string>TRACE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QTextBrowser" name="history"/>
      </item>
      <item row="7" column="1" rowspan="2">
       <widget class="QTextEdit" name="input"/>
      </item>
      <item row="7" column="4" rowspan="2">
       <widget class="QLineEdit" name="id_result"/>
      </item>
      <item row="10" column="1" colspan="3">
       <widget class="QTextBrowser" name="warning"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
